getssl

```diff
@@ -1089,15 +1091,8 @@ done
 if [ -f "$DOMAIN_DIR/${DOMAIN}.csr" ]; then
   debug "domain csr exists at - $DOMAIN_DIR/${DOMAIN}.csr"
   # check all domains in config are in csr
-  alldomains=$(echo "$DOMAIN,$SANS" | tr -d " " |tr , '\n')
-  # check domain name in CN
-  domain_in_cn=$(openssl req -noout -text -in "$DOMAIN_DIR/${DOMAIN}.csr" |grep -o "CN=.*" | tr -d "CN=" | awk -F"/" '{print $1}')
-  # check domain names in SAN
-  domains_in_san=$(openssl req -noout -text -in "$DOMAIN_DIR/${DOMAIN}.csr" |grep "DNS:.*" |tr -d "DNS:" |tr -d " " |tr , '\n')
-  #combing SN and SAN names
-  domains_total=$(echo "$domain_in_cn"; echo "${domains_in_san}")
-  # remove blank lines (if only CN or SAN were used ) 
-  domains_in_csr=$(echo "$domains_total" | grep -v "^$")
+  alldomains=$(echo "$DOMAIN,$SANS" | sed -e 's/ //g; s/,/\n/g' | sort -u)
```

Comment:
For the manual page - https://developer.apple.com/legacy/library/documentation/Darwin/Reference/ManPages/man1/sed.1.html